butchland/my-ai4d-research-bank
# Fix/disable current actions

## PR Title Convention

Copy paste to PR title, remove this section after complying

- <pr-type>: pr title

example:

- `feat: add button`
- `chore: sort folders`
- `tests: add tests`

## Extra details

Add any extra details, or provide context to the person who will be reviewing the PR or future
developers who may need to use the PR as context

## Changelog

Add a summary of the things that were changed

- [x] Add toggle filter
- [ ] Optional: add fitler sort

## How to test

Add a numbered list of steps to make in order to verify that the feature is working. Ideally add a video of you
running the new features you have added. You can use a capturing tool like [ShareX](https://getsharex.com)

0. Do this setup step in case
1. Do certain actions
2. Assert that feature does thing
